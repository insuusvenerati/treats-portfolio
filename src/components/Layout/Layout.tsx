/* eslint-disable react/no-danger */
import { Link } from 'gatsby';
import { HelmetDatoCms } from 'gatsby-source-datocms';
import React, { useState } from 'react';
import 'react-image-lightbox/style.css';
import '../../styles/fonts.css';
import useLayoutData from '../../hooks/useLayoutData';
import ImageTooltip from '../ImageTooltip/ImageTooltip';
import Img from 'gatsby-image';

const Layout: React.FC = ({ children }) => {
  const [showMenu, setShowMenu] = useState(false);
  const {
    datoCmsSite,
    datoCmsHome,
    allDatoCmsSocialProfile,
    sidebar,
    catImages: { nodes: catImages },
  } = useLayoutData();
  const [catImage, setCatImage] = useState(catImages[0].fixed);

  const generateCatImageHandler = () => {
    const randomCatImage = catImages[Math.floor(Math.random() * catImages.length)].fixed;
    setCatImage(randomCatImage);
  };

  return (
    <div className={`container ${showMenu ? 'is-open' : ''}`}>
      <HelmetDatoCms favicon={datoCmsSite.faviconMetaTags} seo={datoCmsHome.seoMetaTags} />

      <div data-cy="sidebar-container" className="container__sidebar">
        <ImageTooltip generateRandomCatImageHandler={generateCatImageHandler} catImage={catImage} />
        <div style={{ backgroundImage: `url(${sidebar.nodes[0].fixed.src})` }} className={`sidebar`}>
          <h6 style={{ fontFamily: 'Montserrat' }} className="sidebar__title">
            <Link data-cy="site-title" to="/">
              {datoCmsSite.globalSeo.siteName}
            </Link>
          </h6>
          <div
            className="sidebar__intro"
            dangerouslySetInnerHTML={{
              __html: datoCmsHome.introTextNode.childMarkdownRemark.html,
            }}
          />
          <ul data-cy="sidebar-menu" className="sidebar__menu">
            <li>
              <Link activeStyle={{ color: '#5d6b5b' }} to="/">
                bg & illustration
              </Link>
            </li>
            <li>
              <Link activeStyle={{ color: '#5d6b5b' }} to="/sketchbook">
                sketchbook
              </Link>
            </li>
            <li>
              <Link activeStyle={{ color: '#5d6b5b' }} to="/about">
                about & contact
              </Link>
            </li>
          </ul>
          <div data-cy="icon-row" className="centered">
            {allDatoCmsSocialProfile.edges.map(({ node: profile }) => (
              <a
                className={`social grow`}
                key={profile.profileType}
                style={{
                  cursor: 'pointer',
                }}
                href={profile.url}
                target="_blank"
                rel="noreferrer"
              >
                <Img alt={profile.profileType} fluid={profile.icon.fluid} />
              </a>
            ))}
            <a style={{ width: '55px' }}>
              <span
                data-tip
                data-event="click"
                data-for="catGenerator"
                className="social social--cat grow"
                style={{ cursor: 'pointer' }}
              />
            </a>
          </div>
        </div>
      </div>
      <div className="container__body">
        <div className="container__mobile-header">
          <div className="mobile-header">
            <div className="mobile-header__menu">
              <a
                href="#"
                onClick={(e) => {
                  e.preventDefault();
                  setShowMenu(!showMenu);
                }}
              />
            </div>
            <div className="mobile-header__logo">
              <Link to="/">{datoCmsSite.globalSeo.siteName}</Link>
            </div>
          </div>
        </div>
        {children}
      </div>
    </div>
  );
};

export default Layout;
