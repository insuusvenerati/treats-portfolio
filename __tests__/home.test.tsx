import renderer from 'react-test-renderer';
import data from '../__mocks__/layoutData';
import Layout from '../src/components/Layout';

// eslint-disable-next-line @typescript-eslint/no-unsafe-return
jest.mock('crypto', () => ({ randomBytes: (num) => new Array(num).fill(0) }));

it('Should match snapshot', () => {
  const tree = renderer.create(<Layout data={data} />).toJSON();
  expect(tree).toMatchSnapshot();
});
