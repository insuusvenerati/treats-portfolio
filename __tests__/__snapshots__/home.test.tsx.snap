// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match snapshot 1`] = `
<div
  className="container "
>
  <div
    className="container__sidebar"
    data-cy="sidebar-container"
  >
    <div
      className="__react_component_tooltip t00000000-0000-4000-8000-000000000000 place-top type-dark"
      data-id="tooltip"
      id="catGenerator"
    >
      <style
        aria-hidden="true"
        dangerouslySetInnerHTML={
          Object {
            "__html": "
  	.t00000000-0000-4000-8000-000000000000 {
	    color: #fff;
	    background: #222;
	    border: 1px solid transparent;
  	}

  	.t00000000-0000-4000-8000-000000000000.place-top {
        margin-top: -10px;
    }
    .t00000000-0000-4000-8000-000000000000.place-top::before {
        border-top: 8px solid transparent;
    }
    .t00000000-0000-4000-8000-000000000000.place-top::after {
        border-left: 8px solid transparent;
        border-right: 8px solid transparent;
        bottom: -6px;
        left: 50%;
        margin-left: -8px;
        border-top-color: #222;
        border-top-style: solid;
        border-top-width: 6px;
    }

    .t00000000-0000-4000-8000-000000000000.place-bottom {
        margin-top: 10px;
    }
    .t00000000-0000-4000-8000-000000000000.place-bottom::before {
        border-bottom: 8px solid transparent;
    }
    .t00000000-0000-4000-8000-000000000000.place-bottom::after {
        border-left: 8px solid transparent;
        border-right: 8px solid transparent;
        top: -6px;
        left: 50%;
        margin-left: -8px;
        border-bottom-color: #222;
        border-bottom-style: solid;
        border-bottom-width: 6px;
    }

    .t00000000-0000-4000-8000-000000000000.place-left {
        margin-left: -10px;
    }
    .t00000000-0000-4000-8000-000000000000.place-left::before {
        border-left: 8px solid transparent;
    }
    .t00000000-0000-4000-8000-000000000000.place-left::after {
        border-top: 5px solid transparent;
        border-bottom: 5px solid transparent;
        right: -6px;
        top: 50%;
        margin-top: -4px;
        border-left-color: #222;
        border-left-style: solid;
        border-left-width: 6px;
    }

    .t00000000-0000-4000-8000-000000000000.place-right {
        margin-left: 10px;
    }
    .t00000000-0000-4000-8000-000000000000.place-right::before {
        border-right: 8px solid transparent;
    }
    .t00000000-0000-4000-8000-000000000000.place-right::after {
        border-top: 5px solid transparent;
        border-bottom: 5px solid transparent;
        left: -6px;
        top: 50%;
        margin-top: -4px;
        border-right-color: #222;
        border-right-style: solid;
        border-right-width: 6px;
    }
  ",
          }
        }
      />
      <div
        className=" gatsby-image-wrapper"
        style={
          Object {
            "borderRadius": "7px",
            "display": "inline-block",
            "height": 169,
            "overflow": "hidden",
            "position": "relative",
            "width": 149,
            "zIndex": 2,
          }
        }
      >
        <picture>
          <source
            srcSet="https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=1&w=149 1x,
https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=1.5&w=149 1.5x,
https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=2&w=149 2x,
https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=3&w=149 3x"
          />
          <img
            alt=""
            height={169}
            loading="lazy"
            onLoad={[Function]}
            src="https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&w=149"
            srcSet="https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=1&w=149 1x,
https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=1.5&w=149 1.5x,
https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=2&w=149 2x,
https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=3&w=149 3x"
            style={
              Object {
                "height": "100%",
                "left": 0,
                "objectFit": "cover",
                "objectPosition": "center",
                "opacity": 0,
                "position": "absolute",
                "top": 0,
                "transition": "opacity 500ms",
                "width": "100%",
              }
            }
            width={149}
          />
        </picture>
        <noscript
          dangerouslySetInnerHTML={
            Object {
              "__html": "<picture><source srcset=\\"https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=1&w=149 1x,
https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=1.5&w=149 1.5x,
https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=2&w=149 2x,
https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=3&w=149 3x\\" /><img loading=\\"lazy\\" width=\\"149\\" height=\\"169\\" srcset=\\"https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=1&w=149 1x,
https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=1.5&w=149 1.5x,
https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=2&w=149 2x,
https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&dpr=3&w=149 3x\\" src=\\"https://www.datocms-assets.com/23328/1601060463-8.png?auto=format&w=149\\" alt=\\"\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
            }
          }
        />
      </div>
    </div>
    <div
      className="sidebar"
      style={
        Object {
          "backgroundImage": "url(https://www.datocms-assets.com/23328/1594784357-sidebarbg.png?auto=format&w=225)",
        }
      }
    >
      <h6
        className="sidebar__title"
        style={
          Object {
            "fontFamily": "Montserrat",
          }
        }
      >
        <a
          data-cy="site-title"
          href="/"
        >
          laura norwood
        </a>
      </h6>
      <div
        className="sidebar__intro"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<p>visdev • bg painting • illustration</p>",
          }
        }
      />
      <ul
        className="sidebar__menu"
        data-cy="sidebar-menu"
      >
        <li>
          <a
            href="/"
          >
            bg & illustration
          </a>
        </li>
        <li>
          <a
            href="/sketchbook"
          >
            sketchbook
          </a>
        </li>
        <li>
          <a
            href="/about"
          >
            about & contact
          </a>
        </li>
      </ul>
      <div
        className="centered"
        data-cy="icon-row"
      >
        <a
          className="social grow"
          href="https://twitter.com/Treatscraft"
          rel="noreferrer"
          style={
            Object {
              "cursor": "pointer",
            }
          }
          target="_blank"
        >
          <div
            className=" gatsby-image-wrapper"
            style={
              Object {
                "maxHeight": null,
                "maxWidth": null,
                "overflow": "hidden",
                "position": "relative",
              }
            }
          >
            <div
              aria-hidden={true}
              style={
                Object {
                  "paddingBottom": "94.23076923076923%",
                  "width": "100%",
                }
              }
            />
            <picture>
              <source
                sizes="(max-width: 60px) 100vw, 60px"
                srcSet="https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.08&fm=webp&w=208 15w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.15&fm=webp&w=208 30w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.29&fm=webp&w=208 60w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.44&fm=webp&w=208 90w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.58&fm=webp&w=208 120w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.87&fm=webp&w=208 180w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=1&fm=webp&w=208 208w"
              />
              <img
                alt="Twitter"
                loading="lazy"
                onLoad={[Function]}
                sizes="(max-width: 60px) 100vw, 60px"
                src="https://www.datocms-assets.com/23328/1595107438-twitter.png?fm=webp"
                srcSet="https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.08&fm=webp&w=208 15w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.15&fm=webp&w=208 30w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.29&fm=webp&w=208 60w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.44&fm=webp&w=208 90w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.58&fm=webp&w=208 120w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.87&fm=webp&w=208 180w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=1&fm=webp&w=208 208w"
                style={
                  Object {
                    "height": "100%",
                    "left": 0,
                    "objectFit": "cover",
                    "objectPosition": "center",
                    "opacity": 0,
                    "position": "absolute",
                    "top": 0,
                    "transition": "opacity 500ms",
                    "width": "100%",
                  }
                }
              />
            </picture>
            <noscript
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<picture><source srcset=\\"https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.08&fm=webp&w=208 15w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.15&fm=webp&w=208 30w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.29&fm=webp&w=208 60w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.44&fm=webp&w=208 90w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.58&fm=webp&w=208 120w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.87&fm=webp&w=208 180w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=1&fm=webp&w=208 208w\\" sizes=\\"(max-width: 60px) 100vw, 60px\\" /><img loading=\\"lazy\\" sizes=\\"(max-width: 60px) 100vw, 60px\\" srcset=\\"https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.08&fm=webp&w=208 15w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.15&fm=webp&w=208 30w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.29&fm=webp&w=208 60w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.44&fm=webp&w=208 90w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.58&fm=webp&w=208 120w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=0.87&fm=webp&w=208 180w,
https://www.datocms-assets.com/23328/1595107438-twitter.png?dpr=1&fm=webp&w=208 208w\\" src=\\"https://www.datocms-assets.com/23328/1595107438-twitter.png?fm=webp\\" alt=\\"Twitter\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
                }
              }
            />
          </div>
        </a>
        <a
          className="social grow"
          href="https://www.instagram.com/ljanorwood/"
          rel="noreferrer"
          style={
            Object {
              "cursor": "pointer",
            }
          }
          target="_blank"
        >
          <div
            className=" gatsby-image-wrapper"
            style={
              Object {
                "maxHeight": null,
                "maxWidth": null,
                "overflow": "hidden",
                "position": "relative",
              }
            }
          >
            <div
              aria-hidden={true}
              style={
                Object {
                  "paddingBottom": "94.68599033816426%",
                  "width": "100%",
                }
              }
            />
            <picture>
              <source
                sizes="(max-width: 60px) 100vw, 60px"
                srcSet="https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.08&fm=webp&w=207 15w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.15&fm=webp&w=207 30w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.29&fm=webp&w=207 60w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.44&fm=webp&w=207 90w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.58&fm=webp&w=207 120w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.87&fm=webp&w=207 180w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=1&fm=webp&w=207 207w"
              />
              <img
                alt="Instagram"
                loading="lazy"
                onLoad={[Function]}
                sizes="(max-width: 60px) 100vw, 60px"
                src="https://www.datocms-assets.com/23328/1595107432-insta.png?fm=webp"
                srcSet="https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.08&fm=webp&w=207 15w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.15&fm=webp&w=207 30w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.29&fm=webp&w=207 60w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.44&fm=webp&w=207 90w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.58&fm=webp&w=207 120w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.87&fm=webp&w=207 180w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=1&fm=webp&w=207 207w"
                style={
                  Object {
                    "height": "100%",
                    "left": 0,
                    "objectFit": "cover",
                    "objectPosition": "center",
                    "opacity": 0,
                    "position": "absolute",
                    "top": 0,
                    "transition": "opacity 500ms",
                    "width": "100%",
                  }
                }
              />
            </picture>
            <noscript
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<picture><source srcset=\\"https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.08&fm=webp&w=207 15w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.15&fm=webp&w=207 30w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.29&fm=webp&w=207 60w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.44&fm=webp&w=207 90w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.58&fm=webp&w=207 120w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.87&fm=webp&w=207 180w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=1&fm=webp&w=207 207w\\" sizes=\\"(max-width: 60px) 100vw, 60px\\" /><img loading=\\"lazy\\" sizes=\\"(max-width: 60px) 100vw, 60px\\" srcset=\\"https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.08&fm=webp&w=207 15w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.15&fm=webp&w=207 30w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.29&fm=webp&w=207 60w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.44&fm=webp&w=207 90w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.58&fm=webp&w=207 120w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=0.87&fm=webp&w=207 180w,
https://www.datocms-assets.com/23328/1595107432-insta.png?dpr=1&fm=webp&w=207 207w\\" src=\\"https://www.datocms-assets.com/23328/1595107432-insta.png?fm=webp\\" alt=\\"Instagram\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
                }
              }
            />
          </div>
        </a>
        <a
          className="social grow"
          href="mailto:norwood.lja@gmail.com"
          rel="noreferrer"
          style={
            Object {
              "cursor": "pointer",
            }
          }
          target="_blank"
        >
          <div
            className=" gatsby-image-wrapper"
            style={
              Object {
                "maxHeight": null,
                "maxWidth": null,
                "overflow": "hidden",
                "position": "relative",
              }
            }
          >
            <div
              aria-hidden={true}
              style={
                Object {
                  "paddingBottom": "94.23076923076923%",
                  "width": "100%",
                }
              }
            />
            <picture>
              <source
                sizes="(max-width: 60px) 100vw, 60px"
                srcSet="https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.08&fm=webp&w=208 15w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.15&fm=webp&w=208 30w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.29&fm=webp&w=208 60w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.44&fm=webp&w=208 90w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.58&fm=webp&w=208 120w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.87&fm=webp&w=208 180w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=1&fm=webp&w=208 208w"
              />
              <img
                alt="Email"
                loading="lazy"
                onLoad={[Function]}
                sizes="(max-width: 60px) 100vw, 60px"
                src="https://www.datocms-assets.com/23328/1595107435-mail.png?fm=webp"
                srcSet="https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.08&fm=webp&w=208 15w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.15&fm=webp&w=208 30w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.29&fm=webp&w=208 60w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.44&fm=webp&w=208 90w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.58&fm=webp&w=208 120w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.87&fm=webp&w=208 180w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=1&fm=webp&w=208 208w"
                style={
                  Object {
                    "height": "100%",
                    "left": 0,
                    "objectFit": "cover",
                    "objectPosition": "center",
                    "opacity": 0,
                    "position": "absolute",
                    "top": 0,
                    "transition": "opacity 500ms",
                    "width": "100%",
                  }
                }
              />
            </picture>
            <noscript
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<picture><source srcset=\\"https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.08&fm=webp&w=208 15w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.15&fm=webp&w=208 30w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.29&fm=webp&w=208 60w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.44&fm=webp&w=208 90w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.58&fm=webp&w=208 120w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.87&fm=webp&w=208 180w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=1&fm=webp&w=208 208w\\" sizes=\\"(max-width: 60px) 100vw, 60px\\" /><img loading=\\"lazy\\" sizes=\\"(max-width: 60px) 100vw, 60px\\" srcset=\\"https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.08&fm=webp&w=208 15w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.15&fm=webp&w=208 30w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.29&fm=webp&w=208 60w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.44&fm=webp&w=208 90w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.58&fm=webp&w=208 120w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=0.87&fm=webp&w=208 180w,
https://www.datocms-assets.com/23328/1595107435-mail.png?dpr=1&fm=webp&w=208 208w\\" src=\\"https://www.datocms-assets.com/23328/1595107435-mail.png?fm=webp\\" alt=\\"Email\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
                }
              }
            />
          </div>
        </a>
        <a
          style={
            Object {
              "width": "55px",
            }
          }
        >
          <span
            className="social social--cat grow"
            data-event="click"
            data-for="catGenerator"
            data-tip="true"
            style={
              Object {
                "cursor": "pointer",
              }
            }
          />
        </a>
      </div>
    </div>
  </div>
  <div
    className="container__body"
  >
    <div
      className="container__mobile-header"
    >
      <div
        className="mobile-header"
      >
        <div
          className="mobile-header__menu"
        >
          <a
            href="#"
            onClick={[Function]}
          />
        </div>
        <div
          className="mobile-header__logo"
        >
          <a
            href="/"
          >
            laura norwood
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
